src/garage/sampler/utils.py

```diff
@@ -24,7 +35,13 @@ def rollout(env,
         env.render()
     while path_length < max_path_length:
         a, agent_info = agent.get_action(o)
-        next_o, r, d, env_info = env.step(a)
+
+        if deterministic == False:
+            action = a
+        else:
+            action = agent_info['mean']
+
+        next_o, r, d, env_info = env.step(action)
```

Comment:
Seems like this block of code can be simplified to make it a bit more pythonic.

```python
if deterministic:
    a = agent_info['mean']

next_o, r, d, env_info = env.step(a)
```

One possible benefit of this change is that it fixes what seems like a bug below, where the "old" action `a` is appended to the action space instead of the "new" one.